Formula/pulumi.rb

```diff
@@ -17,28 +17,21 @@ class Pulumi < Formula
   depends_on "go" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
-    ENV["GO111MODULE"] = "on"
+    cd "./sdk" do
+      system "go", "mod", "download"
+    end
+    cd "./pkg" do
+      system "go", "mod", "download"
+    end
 
-    dir = buildpath/"src/github.com/pulumi/pulumi"
-    dir.install buildpath.children
+    system "make", "brew"
 
-    cd dir do
-      cd "./sdk" do
-        system "go", "mod", "download"
-      end
-      cd "./pkg" do
-        system "go", "mod", "download"
-      end
-      system "make", "brew"
-      bin.install Dir["#{buildpath}/bin/*"]
-      prefix.install_metafiles
+    bin.install Dir["#{ENV["GOPATH"]}/bin/pulumi*"]
```

Comment:
This shouldn't work right?

Reply:
I didn't add this - this wasn't in my original PR - this was a force push by @dawidd6

My original PR had 2 values change and nothing else!

Reply:
Why not?

Reply:
Cause GOPATH isn't set anywhere?

Reply:
It is:
https://github.com/Homebrew/brew/blob/778bd543c72df0fbee3e7562a3bc89fc6d8b7f32/Library/Homebrew/formula.rb#L2142